package.ts

```diff
@@ -73,6 +73,20 @@ const pkg = {
           default: true,
           description: 'Enables or disables the Dance key bindings.',
         },
+        'dance.normalMode.lineNumbers': {
+          type: 'integer',
+          enum: ['off', 'on', 'relative', 'inherit'],
+          default: 'relative',
+          description: 'What line numbering to use in normal mode',
+          enumDescriptions: ['No line numbers', 'Standard line numbering', 'Relative line numbers', 'Use the value from the editor settings'],
+        },
+        'dance.insertMode.lineNumbers': {
+          type: 'integer',
+          enum: ['off', 'on', 'relative', 'inherit'],
+          default: 'inherit',
+          description: 'What line numbering to use in insert mode',
+          enumDescriptions: ['No line numbers', 'Standard line numbering', 'Relative line numbers', 'Use the value from the editor settings'],
```

Comment:
Thanks! For consistency, can you change the descriptions to:
- No line numbers.
- Absolute line numbers.
- Relative line numbers.
- Inherit from `editor.lineNumbers`.

To get closer to VS Code, we could also set the description to "Controls the display of line numbers in {insert,normal} mode."

Do note that the descriptions should end with periods.